Update collate field

teunbrand opened the pull request:
Should fix #10, I think.

teunbrand committed:
Update collate field

clauswilke commented:
Sorry to be picky: Any idea why the diff shows complete replacement of the `DESCRIPTION` file, instead of just addition of one line? Is there a newline issue? (I usually see this happen when all newlines in a file have been changed from `\n` to `\r\n` or vice versa, which happens e.g. if the file was edited on Windows while using a broken git setup.)

teunbrand committed:
Replace windows newline

teunbrand committed:
Manually change all newlines on github

teunbrand commented:
Yes that might exactly be what happened. I tried replacing all newlines manually on github instead of windows; did that help anything? Thanks for the explanation by the way, I didn't know that git on widows required some extra configuration.

clauswilke commented:
For more details see here:
https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings

It doesn't look fixed yet. When you look at the diff, you shouldn't see all lines removed and then re-added. Not sure why this is popping up now, though. It's entirely possible that the `DESCRIPTION` currently in the repo has broken line endings, in which case changing all lines would be correct.

teunbrand committed:
Commit from reconfigured git

teunbrand committed:
Empty commit?

Merge branch 'master' of https://github.com/teunbrand/ggtextures

# Conflicts:
#	DESCRIPTION

teunbrand commented:
Ah yes I think I managed now. Thank you for your patience, Claus!

clauswilke commented:
Thanks!
